docs/en/observability/synthetics-troubleshooting.asciidoc

```diff
@@ -65,6 +65,44 @@ in the {synthetics-app}.
 
 To fix this, make sure there is an agent configured to run against the agent policy.
 
+[discrete]
+[[synthetics-troubleshooting-no-direct-es-connection]]
+== No results from a monitor
+
+If you have configured a monitor but don't see any results for that monitor in the {synthetics-app}, whether running them from Elastic's global managed testing infrastructure or from {private-location}s, ensure Synthetics has a direct connection to {es}.
+
+Do not configure any ingest pipelines or output via Logstash as this will prevent Synthetics from working properly and is not <<synthetics-support-matrix,supported>>.
+
+[discrete]
+[[synthetics-troubleshooting-missing-browser-schedules]]
+== Browser monitor configured to run on a {private-location} not running to schedule
+
+If you have browser monitors configured to run on a {private-location} but notice one or more of them are not running as scheduled, this could be because:
+
+* The time it takes for your monitor to run is longer than the frequency you have set
+* There may be too many monitors trying to run concurrently, causing some of them to skip their scheduled run
+
+You may also see a message in the logs such as `2 tasks have missed their schedule deadlines by more than 1 second in the last 15s`. These will be visible from inside the Agent diagnostic ZIP file, and the numbers and time periods may be different in your logs.
+
+Start by identifying the cause of the issue. First, check if the time it takes the monitor to run is less than the scheduled frequency:
+
+1. Go to the Synthetics app.
+2. Click the monitor, then click **Go to monitor**.
+3. Go to the https://www.elastic.co/guide/en/observability/current/synthetics-analyze.html#synthetics-analyze-individual-monitors-overview[Overview tab] to see the _Avg. duration_. You can also view the duration for individual runs in the https://www.elastic.co/guide/en/observability/current/synthetics-analyze.html#synthetics-analyze-individual-monitors-history[History tab].
+4. Compare the duration to the scheduled frequency. If the duration is _greater than_ the scheduled frequency, for example if the monitor that takes 90 seconds to run and its scheduled frequency is 1 minute, the next scheduled run will not occur because the current one is still running so you may see results for every other scheduled run.
+
+To fix this, you can either:
+
+* Change the frequency so the monitor runs less often.
+* Refactor the monitor so it can run in a shorter amount of time.
```

Comment:
```suggestion
4. Compare the duration to the scheduled frequency. If the duration is _greater than_ the scheduled frequency, for example if the monitor that takes 90 seconds to run and its scheduled frequency is 1 minute, the next scheduled run will not occur because the current one is still running so you may see results for every other scheduled run.
+
To fix this, you can either:
+
* Change the frequency so the monitor runs less often.
* Refactor the monitor so it can run in a shorter amount of time.
```